tests/models.py

```diff
@@ -41,15 +41,15 @@ class ModelWithSlugField2(models.Model):
 
 class MenManager(Manager):
 
-    def get_query_set(self):
-        query_set = super(MenManager, self).get_query_set()
+    def get_queryset(self):
+        query_set = super(MenManager, self).get_queryset()
         return query_set.filter(gender=1)
 
 
 class WomenManager(Manager):
 
-    def get_query_set(self):
-        query_set = super(WomenManager, self).get_query_set()
+    def get_queryset(self):
```

Comment:
I took this change from the 1.7 compatibility PR. The tests do not fail in <1.7 Django, even though the function must be named `get_query_set` there. So this clearly is never run by the tests - I wonder why the code is here then?!